src/index.js

```diff
@@ -115,19 +115,19 @@ export default class Shape {
     return this.areas().reduce((a, b) => a + b);
   }
   
-  perimiter(index) {
+  perimeter(index) {
     const path = this.paths[index];
-    const perimiter = ClipperLib.Clipper.PerimiterOfPath(path, this.closed);
-    return perimiter;
+    const perimeter = ClipperLib.Clipper.PerimeterOfPath(path, this.closed);
+    return perimeter;
   }
   
-  perimiters() {
-    return this.paths.map(path => ClipperLib.Clipper.PerimiterOfPath(path, this.closed))
+  perimeters() {
+    return this.paths.map(path => ClipperLib.Clipper.PerimeterOfPath(path, this.closed))
```

Comment:
Ah, an easy mistake for me to make at this point as my linters all default to [Standard](http://standardjs.com/) these days.